app/src/main/java/org/jellyfin/androidtv/ui/shared/ToolbarView.kt

```diff
@@ -7,10 +7,17 @@ import android.view.ViewGroup
 import android.widget.FrameLayout
 import kotlinx.android.synthetic.main.view_toolbar.view.*
 import org.jellyfin.androidtv.R
+import org.jellyfin.androidtv.preference.UserPreferences
+import org.jellyfin.androidtv.preference.UserPreferences.Companion.clockBehavior
+import org.jellyfin.androidtv.preference.constant.ClockBehavior
+import org.koin.java.KoinJavaComponent.get
 
 class ToolbarView(context: Context, attrs: AttributeSet) : FrameLayout(context, attrs) {
 	init {
 		inflate(context, R.layout.view_toolbar, this)
+		val clockBehavior = get(UserPreferences::class.java)[clockBehavior]
```

Comment:
```suggestion
		val clockBehavior = get(UserPreferences::class.java)[UserPreferences.Companion.clockBehavior]
```

Although the `UserPreferences.Companion.` doesn't look that great, it's easier to read since your variable has the exact same name. Could also just rename the variable.